ci/jest-reporter.js

```diff
@@ -1,47 +1,85 @@
+const { generateDiscordCloudwatchLogUrls, listECSTasks, sendDiscordNotification } = require('./helpers')
+
 const { BaseReporter } = require('@jest/reporters')
 const child_process = require('child_process')
 
+async function listArntasks() {
+  taskArns = await listECSTasks()  // like taskArns = ['arn:aws:ecs:*********:************:task/ceramic-dev-tests/2466935a544f47ec9a1c3d8add235c84']
+  return taskArns
+}
```

Comment:
I see no reference to the taskArns variable until it is assigned here. 🤔  Looking at this it seems like it should error but I'm guessing JS is doing something that I don't recognize.

In any case the return is better imo.
```suggestion
async function listArntasks() {
  return await listECSTasks()  // like taskArns = ['arn:aws:ecs:*********:************:task/ceramic-dev-tests/2466935a544f47ec9a1c3d8add235c84']
}
```